infrastructure/terraform/variables.tf

```diff
@@ -36,8 +36,33 @@ variable "protect_keyvault" {
   default     = true
 }
 
+variable "postgres_backup_retention_days" {
+  description = "The number of days to retain backups for the PostgreSQL Flexible Server."
+  type        = number
+  default     = 30
```

Comment:
Why do we set the default to 30 and override with 7? Not sure how long we should be keeping them either. Do we have programme NFRs for this stuff?